src/UI/Factory.php

```diff
@@ -336,4 +337,27 @@ public function panel();
 	 **/
 	public function modal();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     Popovers are used to display secondary information to a component.
+	 *   composition: >
+	 *     Popovers consist of a short title and text.
+	 *   effect: >
+	 *     Popovers are typically shown by clicking or hovering over a component. The position
+	 *     of the Popover is calculated automatically be default. However, it is possible to
+	 *     specify if the popover appears at the top, right, bottom or left of the triggerer component.
+	 * context: >
+	 * rules:
+	 *   usage:
+	 *     1: Popovers MUST contain a title and a text.
```

Comment:
I quickly checked what happens on the current implementation if you pass en empty string as title: The title box is not rendered at all. So the title _could_ be made optional, e.g. one would provide a title via `withTitle()`